### PR TITLE
CIF-736 - Rename CIF to CIF REST

### PR DESCRIPTION
**Pull request by herzog31**
## Description
Renaming CIF to CIF REST to better distinguish between REST and GraphQL APIs.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code passes the code style as defined by ESLint.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,39 +4,39 @@
...
 
 Note that the build actually generates these files under the `src/main/resources/generated` folder, but this folder is not committed on the git repository. The `maven release` process actually copies the generated files to the folders mentioned above so that we only have released files in the repository.
 
 ### GraphQL endpoint and specification
 
-Our Swagger API also defines the `/graphql` HTTP endpoint for the CIF [GraphQL](https://graphql.org/) API. Because Swagger is only designed to document a REST API, the GraphQL API and schema has its own [documentation](http://opensource.adobe.com/commerce-cif-api/graphql/index.html). Check this documentation to find out about the operations currently supported by the CIF GraphQL API.
+Our Swagger API also defines the `/graphql` HTTP endpoint for the CIF REST [GraphQL](https://graphql.org/) API. Because Swagger is only designed to document a REST API, the GraphQL API and schema has its own [documentation](http://opensource.adobe.com/commerce-cif-api/graphql/index.html). Check this documentation to find out about the operations currently supported by the CIF REST GraphQL API.
```
Comment (cjelger) [inline README.md]: I guess here having REST with GraphQL is totally misleading, but fine if we anyway deprecate the GraphQL part and remove it from the documentation.

Comment (mhaack) [reply README.md]: This section should be remove anyway, we have CIF-737 for that.

**Comment by mhaack** [comment]
@cjelger the is a good question Aditi mentioned only "CIF REST", did not say anything about "CIF Cloud REST".

**Comment by herzog31** [comment]
@cjelger I asked for clarification on the issue and will update it accordingly then.

**Comment by herzog31** [comment]
@cjelger @mhaack Product management decided on "CIF Rest". I updated the PR accordingly and resolved conflicts.

**Pull request merged**